packages/hash/api/package.json

```diff
@@ -60,7 +60,7 @@
     "multer": "1.4.4",
     "nanoid": "3.1.31",
     "nocache": "^3.0.1",
-    "node-fetch": "^2.6.7",
+    "node-fetch": "3.2.0",
```

Comment:
We still end up with 2.6.7 because of `package.json` resolutions:

```
  "resolutions": {
    "node-fetch": "^2.6.7"
  },
```

See `yarn.lock` diff.

I tried removing `resolutions` a couple of days ago, but I could not because we still have several pretty dated dependencies that pull 2.6.1 :(

What features of `3.2.1`  were you going to use?

Reply:
Not planning to use the higher version, this is what `yarn fix:dependency-version-consistency` came back with. How would i fix the yarn deduplicate errors [seen here](https://github.com/hashintel/hash/runs/5093114611?check_suite_focus=true) if i roll back?

Reply:
Can you try rolling back in `packages/hash/api/package.json`, `packages/hash/shared/package.json` and `packages/hash/integration/package.json`, then run `yarn install`?

Reply:
Ah i see now, i added `packages/hash/integration/package.json` and i must have used 3.2.0 for whatever reason. I think it's better now, thank you!

Reply:
Yep, no changes in `yarn.lock`. Nice!